README.md

```diff
@@ -53,11 +53,58 @@ crud.unflatten_rows(res.rows, res.metadata)
 **Notes:**
 
 * A space should have a format.
-* By default, `bucket_id` is computed as `vshard.router.bucket_id_strcrc32(key)`,
-  where `key` is the primary key value.
-  Custom bucket ID can be specified as `opts.bucket_id` for each operation.
-  For operations that accepts tuple/object bucket ID can be specified as
-  tuple/object field as well as `opts.bucket_id` value.
+
+**Sharding key and bucket id calculation**
+
+*Sharding key* is a set of tuple field values used for calculation *bucket ID*.
+*Sharding key definition* is a set of tuple field names that describe what
+tuple field should be a part of sharding key. *Bucket ID* determines which
+replicaset stores certain data. Function that used for calculation bucket ID is
+named *sharding function*.
+
+By default CRUD calculates bucket ID using primary key and a function
+`vshard.router.bucket_id_strcrc32(key)`, it happen automatically and doesn't
+require any actions from user side. User can calculate bucket ID on outside and
+pass it as an option to CRUD methods that accepts tuple or object (see option
+`bucket_id` below).
+
+In version > 0.8.0 users who don't want to use primary key as a sharding key
```

Comment:
The next crud release will be 0.10.0 (I would prefer 'Starting from 0.10.0' over '> 0.9.0', the latter may be a bit confusing.

Reply:
Updated:

```diff
--- a/README.md
+++ b/README.md
@@ -68,7 +68,7 @@ require any actions from user side. User can calculate bucket ID on outside and
 pass it as an option to CRUD methods that accepts tuple or object (see option
 `bucket_id` below).
 
-In version > 0.8.0 users who don't want to use primary key as a sharding key
+Starting from 0.10.0 users who don't want to use primary key as a sharding key
 may set custom sharding key definition as a part of [DDL
 schema](https://github.com/tarantool/ddl#input-data-format) or insert manually
 to the space `_ddl_sharding_key` (for both cases consider a DDL module
@@ -162,8 +162,7 @@ local object, err = crud.get(space_name, key, opts)
 where:
```